indexer/services/ender/src/scripts/dydx_block_processor_unordered_handlers.sql

```diff
@@ -0,0 +1,69 @@
+
+  Parameters:
+    - block: A 'DecodedIndexerTendermintBlock' converted to JSON format. Conversion to JSON is expected to be done by JSON.stringify.
+
+  Returns:
+    An array containing the results for each event or NULL if this event is not handled by this block processor.
+    See each individual handler function for a description of the the inputs and outputs.
+
+  (Note that no text should exist before the function declaration to ensure that exception line numbers are correct.)
+
+  TODO(IND-514): Remove the batch and sync handlers completely by moving all redis updates into
+  a pipeline similar to how we return kafka events and then batch and emit them.
+*/
+DECLARE
+    USDC_ASSET_ID constant text = '0';
+
+    block_height int = (block->'height')::int;
+    block_time timestamp = (block->>'time')::timestamp;
+    event_ jsonb;
+    rval jsonb[];
+    event_index int;
+    transaction_index int;
+    event_data jsonb;
+BEGIN
+    rval = array_fill(NULL::jsonb, ARRAY[coalesce(jsonb_array_length(block->'events'), 0)]::integer[]);
+
+    /** Note that arrays are 1-indexed in PostgreSQL and empty arrays return NULL for array_length. */
+    FOR i in 1..coalesce(array_length(rval, 1), 0) LOOP
+        event_ = jsonb_array_element(block->'events', i-1);
+        transaction_index = dydx_tendermint_event_to_transaction_index(event_);
+        event_index = (event_->'eventIndex')::int;
+        event_data = event_->'dataBytes';
+        CASE event_->'subtype'
+            WHEN '"order_fill"'::jsonb THEN
+                IF event_data->'order' IS NOT NULL THEN
+                    rval[i] = jsonb_build_object(
+                            'makerOrder',
+                            dydx_order_fill_handler_per_order('makerOrder', block_height, block_time, event_data, event_index, transaction_index, jsonb_array_element_text(block->'txHashes', transaction_index), 'MAKER', 'LIMIT', USDC_ASSET_ID, event_data->>'makerCanceledOrderStatus'),
+                            'order',
+                            dydx_order_fill_handler_per_order('order', block_height, block_time, event_data, event_index, transaction_index, jsonb_array_element_text(block->'txHashes', transaction_index), 'TAKER', 'LIMIT', USDC_ASSET_ID, event_data->>'takerCanceledOrderStatus'));
+                ELSE
+                    rval[i] = jsonb_build_object(
+                            'makerOrder',
+                            dydx_liquidation_fill_handler_per_order('makerOrder', block_height, block_time, event_data, event_index, transaction_index, jsonb_array_element_text(block->'txHashes', transaction_index), 'MAKER', 'LIQUIDATION', USDC_ASSET_ID),
+                            'liquidationOrder',
+                            dydx_liquidation_fill_handler_per_order('liquidationOrder', block_height, block_time, event_data, event_index, transaction_index, jsonb_array_element_text(block->'txHashes', transaction_index), 'TAKER', 'LIQUIDATED', USDC_ASSET_ID));
```

Comment:
nit: Add comments indicating the first case is for normal fills, and the second is for liquidations.